### PR TITLE
Kunlun

### PR DESCRIPTION
**Pull request by leafyoung**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,8 +154,7 @@ class PoolAddress(DTO):
 from credmark.types import Address
 
 def run(self, input):
-   address = Address(wallet_adress)
-   address.checksum
+   address = Address(wallet_adress).checksum
```
Comment (iwyrkore) [inline README.md:157]: I wanted to show you can use the address (lowercase) and also get the checksum. Please undo this.

File: README.md [hunk 2]
```diff
@@ -257,6 +256,21 @@ price = context.run_model('price', token, return_type=credmark.types.Price)
 context.web3 will return a configured web3 instance with the default block set to the block number of context.
 The web3 providers are determined from the environment variables as described in the configuration section above. Currently users will need to use their own alchemy account (or other web3 provider) to access web3 functionality.
 
+### Address
+
+`Address` class is inherited from `str` to help with web3 address conversion. It's highly recommended to use it instead of a baremetal address.
+
+✔️: Address("0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9").checksum
```
Comment (iwyrkore) [inline README.md:263]: Creation is without checksum. You can add more info on getting the checksum.

Comment (leafyoung) [reply README.md:263]: add one more example and description.

✔️: Address("0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9").checksum # checksum version to be used
❌: Address("0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9") # lower case version
❌:"0x7d2768de32b0b80b7a3454c06bdac94a69ddc7a9" # lower case version

File: README.md (hunk 1 above)
Comment (leafyoung) [reply README.md:157]: Add the comments

```
from credmark.types import Address

def run(self, input):
    address = Address(wallet_adress) # address stores the lower case version
    address.checksum # .checksum property returns the checksum
```

**Comment by leafyoung** [comment]
WIP - not ready for merge.

**Comment by iwyrkore** [comment]
Let me know when this is ready to merge?

**Comment by leafyoung** [comment]
This is ready to be merged.

**Pull request merged**